diff --git a/modules/ashell/hm.nix b/modules/ashell/hm.nix
deleted file mode 100644
index 41616ac..0000000
--- a/modules/ashell/hm.nix
+++ /dev/null
@@ -1,35 +0,0 @@
-{ mkTarget, ... }:
-mkTarget {
-  name = "ashell";
-  humanName = "Ashell";
-
-  configElements = [
-    (
-      { colors }:
-      {
-        programs.ashell.settings.appearance = with colors.withHashtag; {
-          background_color = base00;
-          primary_color = base0D;
-          secondary_color = base01;
-          success_color = base0B;
-          danger_color = base09;
-          text_color = base05;
-
-          workspace_colors = [
-            base09
-            base0D
-          ];
-        };
-      }
-    )
-    (
-      { opacity }:
-      {
-        programs.ashell.settings.appearance = {
-          opacity = opacity.desktop;
-          menu.opacity = opacity.desktop;
-        };
-      }
-    )
-  ];
-}
diff --git a/modules/ashell/meta.nix b/modules/ashell/meta.nix
deleted file mode 100644
index f19945e..0000000
--- a/modules/ashell/meta.nix
+++ /dev/null
@@ -1,6 +0,0 @@
-{ lib, ... }:
-{
-  name = "Ashell";
-  homepage = "https://github.com/MalpenZibo/ashell";
-  maintainers = [ lib.maintainers.justdeeevin ];
-}
